assignments/lambda-classes.js

```diff
@@ -1 +1,41 @@
 // CODE here for your Lambda Classes
+
+const Person = new Instructor ({
```

Comment:
class syntax are not hoisted, that means all your variable declarations has to be below the class or else it will not work. Make sure to put all your stuff underneath the class!